ast2vec/repo2/base.py

```diff
@@ -126,22 +127,41 @@ def thread_loop(thread_index):
                             break
                         try:
                             folder, filename, language = task
-                            filepath = os.path.join(folder, filename).encode('utf8')
+                            filepath_naive = os.path.join(folder, filename).encode('utf8')
```

Comment:
It has nothing to do with Ubuntu but rather Linux fs in general or the specific environment. In Linux world, distribution rarely means anything: the packages are built from the same source. The rest is the environment settings